monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -227,7 +227,9 @@ class AppComponent extends AuthComponent {
                                onStatusChange: this.updateStatus,
                                islandMode: this.state.islandMode,
                                defaultReport: this.getDefaultReport(),
-                               sideNavHeader: this.getIslandModeTitle()}
+                               sideNavHeader: this.getIslandModeTitle(),
+                               onLogout: () => {this.auth.logout()
```

Comment:
Not sure this actually invalidates all tokens, but also not sure we need to do that

Reply:
Turns out it doesn't